src/crystal/hasher.cr

```diff
@@ -119,7 +119,10 @@ struct Crystal::Hasher
     t8
   end
 
-  def bytes(value)
+  # Fast hash for `Bytes`.
+  # It treats `Bytes` as binary data, and not as "an array of small numbers".
+  # `Bytes#hash(hasher)` (ie `Slice(UInt8)#hash(hasher)`) calls this method.
```

Comment:
I'm not sure how any of this doc comment is actually relevant to the end user. I'm actually not sure why this method needs a doc comment either @luislavena. This method is exactly the same as using `slice.hash(hasher)`. Perhaps this method would be better `:nodoc:`.

Perhaps we should simply advise people who want to implement a custom `hash` to always use `#hash` on the variable they want to add to the hasher, and remove all the methods on this class apart from `permute(int : UInt64)` and `permute(bytes : Bytes)`.

Let's not implement that suggestion in this PR though, and simply remove this doc comment. my suggestion changes the design of `Hasher` so it's out of scope of this PR.

Reply:
@RX14 my comment was triggered by this line:

```crystal
hasher = hasher.bytes(str.to_slice(len))
```

In `StringPool`. Exposing an undocumented method to other classes.

That was reverted in d0b32b9, which now makes my comment irrelevant.